Mathlib/SetTheory/Ordinal/Notation.lean

```diff
@@ -44,7 +40,7 @@ compile_inductive% ONote
 
 namespace ONote
 
-/-- Notation for 0 -/
+/-- Notation for 0. -/
```

Comment:
I disagree with this sentence: in my book, a docstring should have a period iff it is a complete sentence
Can you revert this one (and similar changes below; please?

Reply:
That makes sense, I'll take it into account moving forward.